pandas/tests/tools/test_to_datetime.py

```diff
@@ -933,6 +933,32 @@ def test_to_datetime_dtarr(self, tz):
         result = to_datetime(arr)
         assert result is arr
 
+    # Doesn't work on Windows since tzpath not set correctly
+    @td.skip_if_windows
+    @pytest.mark.parametrize("arg_class", [Series, Index])
```

Comment:
index_or_series fixture?

Reply:
tbh I'd prefer this, fixtures aren't great for readibility

Reply:
Leaving as is then.